client/client.go

```diff
@@ -93,6 +115,34 @@ func WithHTTPGetter(getter HTTPGetter) Option {
 	}
 }
 
+// WithCacheSize specifies how large of a cache of randomness values should be
+// kept locally. Default 32
+func WithCacheSize(size int) Option {
+	return func(cfg *clientConfig) error {
+		cfg.cacheSize = size
+		return nil
+	}
+}
+
+// WithoutAggregation disables multiple calls to `Watch` being served by
+// a single underlying http poll.
+func WithoutAggregation() Option {
```

Comment:
Why would we ever want to disable this ?

Reply:
🤷‍♂️

Reply:
If we dont even know ourselves, then I would remove the option alltogether.